Makefile

```diff
@@ -4,30 +4,33 @@
 # Example:
 #   make build
 
-
-DIRS=$(shell find . -name project.json -exec dirname {} \;)
 OS=$(shell uname -s)
-LINUX_FRAMEWORK=netcoreapp1.0
-DEFAULT_FRAMEWORK=$(LINUX_FRAMEWORK)
 
+EXAMPLE_DIRS=$(shell find ./example -name *.csproj -exec dirname {} \;)
+TEST_DIRS==$(shell find ./test -name *.csproj -exec dirname {} \;)
+
+LINUX_FRAMEWORK=netcoreapp1.1
+DEFAULT_FRAMEWORK=$(LINUX_FRAMEWORK)
```

Comment:
nit: Might be usable to have `DEFAULT_FRAMEWORK?=..` to allow `make DEFAULT_FRAMEWORK=hehe examples`